hardware/products/ucm/uc5550.md

```diff
@@ -0,0 +1,257 @@
+# UC5550
+![G400D](images/g400d.jpg)
+# Overview
+---
+The UC5550 is a UCM that supports the STM F4 and F7 processor with 16MB of external RAM. It has a built-on STM WiFi module and an Ethernet PHY. It also has 4MB of external flash for large user programs.
+
+# Specifications
+---
+|					|							|
+| -----------------	| -------------------------	|
+| Processor			| STM 32-bit ARM Cortex-M4	|
+| Blah, blah, blah	| Blah, blah, blah			|
```

Comment:
Use TBD, not blah.

Reply:
The blah blah blah was just a filler. If this is something that we want, I can try to fill out the stats.

Reply:
I know, just use TBD as a filler, not blah.

Reply:
ok. I will change it.